### PR TITLE
Clause 11 Security Considerations

### PR DESCRIPTION
**Pull request by cmheazel**
More detailed description of how security is supported and related issues.

### CODE REVIEW

File: core/standard/clause_11_Security_Considerations.adoc [hunk 1]
```diff
@@ -0,0 +1,59 @@
+== Section 11: Security Considerations
+
+[width="90%",cols="2,6a"]
+|===
+|*Requirement {counter:req-id}* |/req/oas30/security +
```
Comment (cportele) [inline core/standard/clause_11_Security_Considerations.adoc]: As this is a requirement in the OAS 3 conformance class, this would need to go to clause 9.

Comment (cportele) [reply core/standard/clause_11_Security_Considerations.adoc]: After looking at this again, I think we need/should do the following:

* Move the OpenAPI specific parts to Clause 9 as part of the OpenAPI requirements class.
* Still add a Clause 11 "Security Considerations" and discuss general Web API vulnerabilities etc. Maybe there is existing text about this somewhere that we can reference. Highlight the importance of HTTPS/TLS. And reference the section in Clause 9.

**Comment by cportele** [comment]
Conference call 10 June 2019: @cmheazel will start the PR over based on the discussion and create a new PR. Use, for example, a reference to chapter 15 of HTTP (https://tools.ietf.org/html/rfc2616#page-150).

**Comment by cmheazel** [comment]
Security Considerations section has undergone a complete re-write.  Comments welcome.

**Comment by cportele** [comment]
+1 from me

**Comment by pvretano** [comment]
@cmheazel looks good to me too ...

**Pull request merged**